Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6396,42 +6396,61 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
     id = "com.google.fonts/check/colorfont_tables",
     rationale = """
         Colr v0 fonts are widely supported in most browsers so they do not require
-        an SVG color table. However, Colr v1 is only well supported in Chrome so
-        we need to add an SVG table to these fonts.
+        an SVG color table. However, Colr v1 is currently not broadly supported
```

Comment:
well... https://caniuse.com/colr-v1 says 72% so I would say that it is broadly supported :)
It's only Safari among the browsers that does not support COLRv1. Plus the Adobe desktop apps..
Maybe reword this as 

> Some environments (e.g. Safari, Adobe apps) do not currently support COLRv1
https://caniuse.com/colr-v1 says 72% so I would say that it is broadly supported :)
It's only Safari among the browsers that does not support COLRv1. Plus the Adobe desktop apps..
Maybe reword this as 

> Some environments (e.g. Safari, Adobe apps) do not currently support COLRv1